java/arcs/core/storage/ActiveStore.kt

```diff
@@ -32,10 +32,17 @@ abstract class ActiveStore<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     open suspend fun idle() = Unit
 
     /**
-     * Registers a [ProxyCallback] with the store and returns a token which can be used to
-     * unregister the callback using [off].
+     * Registers a [ProxyCallback] with the store. A token will either be provided or generated
+     * and returned. The token can be used to unregister the callback using [off].
+     *
+     * A callbackToken will only be provided by a storage component that wraps a store (as is the
+     * case of a Direct Store Muxer wrapping a Direct Store). It is otherwise expected for the store
+     * to generate the callbackToken.
```

Comment:
maybe mention also that if mix the two (ie sometimes you pass the token, sometimes you don't) you can get exceptions if you pass a token that was already used